fe/fe-core/src/main/java/org/apache/doris/catalog/Column.java

```diff
@@ -373,6 +373,9 @@ private void toChildrenThrift(Column column, TColumn tColumn) {
             childrenTColumnType.setIndexLen(children.getOlapColumnIndexSize());
             childrenTColumn.setColumnType(childrenTColumnType);
             childrenTColumn.setIsAllowNull(children.isAllowNull());
+            if (tColumn.getAggregationType() != null) {
+                childrenTColumn.setAggregationType(tColumn.getAggregationType());
```

Comment:
The parent aggregate type is `NONE` by default for array type. If we don't set the aggregate type for the children column, after deserializing in BE, the aggregate type of the children column is `TAggregationType::type::SUM`, because the value of `TAggregationType::type::SUM` is zero.

Reply:
It seems to be a design problem. Please add this comment to the code for notice.

Reply:
Added.